src/Device/Driver/FLARM/Settings.cpp

```diff
@@ -32,6 +53,25 @@ FlarmDevice::RequestSetting(const char *name, OperationEnvironment &env)
   Send(buffer, env);
 }
 
+bool
+FlarmDevice::WaitForSetting(const char *name, unsigned timeout_ms)
+{
+  for (unsigned i = 0; i < timeout_ms / 100; ++i) {
+    if (FlarmDevice::SettingExists(name))
+      return true;
+    Sleep(100);
```

Comment:
Please use std::chrono::milliseconds()

Reply:
Well...`Sleep()` is not taking `std::chrono` as input, just `unsigned`. Only `env.Sleep()` can take `std::chrono`, unless I miss something. The entire codebase shows this.